cluster/node-pools/master-default/userdata.clc.yaml

```diff
@@ -558,6 +558,35 @@ storage:
             volumeMounts:
             - name: config-volume
               mountPath: /etc/nginx
+          - name: skipper-proxy
+            image: registry.opensource.zalan.do/pathfinder/skipper:v0.10.233
+            args:
+            - skipper
+            - -address=:8443
+            - -tls-cert=/etc/kubernetes/ssl/apiserver.pem
+            - -tls-key=/etc/kubernetes/ssl/apiserver-key.pem
+            - -inline-routes
+            - 'z: JWTPayloadAnyKV("iss", "https://identity.zalando.com") -> disableAccessLog() -> "https://127.0.0.1:443"; all: * -> enableAccessLog() -> "https://127.0.0.1:443";'
+            ports:
+            - containerPort: 8443
+            readinessProbe:
+              httpGet:
+                path: /metrics
+                port: 9911
+              initialDelaySeconds: 60
+              timeoutSeconds: 5
+            resources:
+              requests:
+                cpu: 100m
+                memory: 250Mi
```

Comment:
@szuecs Can you recommend a suitable resource request here?

Reply:
SGTM, we have to learn anyways from that